contrib/pyln-testing/pyln/testing/utils.py

```diff
@@ -1020,6 +1020,13 @@ def passes_filters(hmsg, filters):
             out = out[2 + length:]
         return msgs
 
+    def config(self, config_name):
+        try:
+            opt = self.rpc.listconfigs(config_name)
+            return opt[config_name]
```

Comment:
We can also have a default value, that'd also avoid the element not found issue here:

```
return opt.get(config_name, def)
```
Where `def` is just another argument above with `def = None`, so we have the same semantics.